spec/models/post_spec.rb

```diff
@@ -0,0 +1,15 @@
+# == Schema Information
+#
+# Table name: posts
+#
+#  id         :bigint           not null, primary key
+#  body       :text             not null
+#  title      :string           not null
+#  created_at :datetime         not null
+#  updated_at :datetime         not null
+#
+require 'rails_helper'
+
+RSpec.describe Post, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
```

Comment:
We should add tests for this, and also maybe for the views, like request or feature specs.

Reply:
I didn't add tests because I saw there are no test for other things, I thought maybe they weren't needed. I can add tests for this if you want

Reply:
Ideally we should write tests for everything. If not someone's new changes might break some other behavior and we wouldn't know.
But as this is just a playground project, it's not critical if that happens, so I'm Ok.